addons/spreadsheet/static/tests/charts/model/odoo_chart_plugin_test.js

```diff
@@ -582,6 +582,100 @@ QUnit.module("spreadsheet > odoo chart plugin", {}, () => {
         );
     });
 
+    QUnit.test("cumulative line chart with past data before domain period specifying cumulated start as true", async (assert) => {
+        const serverData = getBasicServerData();
```

Comment:
nitpick: can probably extract most of the 30+ lines of setup in constants/helpers. Or just do a single test testing all 3